README.md

```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
-9. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-10. From the release files, copy the `update` folder into your JN installation.
-11. Finally, launch `DDLC.exe`.
+*Use these instructions if you have a BETA version of JN you are updating to a RELEASE (1.0.0 onwards).*
+
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/04:-FAQ#can-i-back-up-my-save-data--how-do-i-find-my-persistent).
+3. Download a fresh copy of DDLC from https://ddlc.moe. **Do not use a Steam installation**.
+4. Extract/unzip DDLC.
+5. Download the release: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
+6. Extract/unzip the release files.
+7. **Delete** the `game` and `lib` folders from your **JN installation**.
+8. **Copy** the `game` and `lib` folders from your **fresh DDLC folder** into your JN installation.
+9. From the release files, copy the `game` folder contents (all of it) into the `game` folder of your JN installation. If asked to replace files, select _Yes_.
+10. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
+11. From the release files, copy the `update` folder into your JN installation.
+12. Finally, launch `DDLC.exe`.
 
 #### Updating from the RELEASE (1.0.0) onwards
 
-*Use these instructions if you have a RELEASE version of JN you are updating to a later version.*
+*Use these instructions if you have a RELEASE version (1.0.0 onwards) of JN you are updating to a later version.*
 
-1. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/)
-2. Download the release files: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
-3. Extract/unzip the new release files.
-4. From the release files, copy the `game` folder contents (all of it) into the `game` folder of your JN installation. If asked to replace files, select _Yes_.
-5. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-6. From the release files, copy the `update` folder contents (all of it) into the `update` folder of your JN installation. If asked to replace files, select _Yes_.
-7. Finally, launch `DDLC.exe`.
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/)
```

Comment:
Why are we linking straight to the git homepage?

Reply:
We should not be - should be pointing to the FAQ segment on persistent backups. Will get that sorted